iree/compiler/Codegen/Utils/Utils.cpp

```diff
@@ -55,6 +55,12 @@ bool isVMVXBackend(IREE::HAL::ExecutableVariantOp variantOp) {
   return variantOp.target().getBackend().getValue() == "vmvx";
 }
 
+bool isVMVXBackend(FuncOp entryPointFn) {
```

Comment:
Nit:  I prefer to make these simple redirects `inline` function and just define it in the header file itself.